env/env.go

```diff
@@ -57,3 +59,11 @@ func GetEnv[T any](key string, options ...Option) T {
 		panic(fmt.Sprintf("unsupported type: %T", result))
 	}
 }
+
+func LoadEnvsFromFile(envFile string) error {
```

Comment:
Let's panic here! Don't be afraid of panics!  They are sometimes good!
This function should be called right at the start of the application, and to keep the client code simple, it's okay to panic here.

So it can be written like this:

```go
func LoadEnvsFromFile(envFile string) {
	if envFile != "" {
		if err := godotenv.Load(envFile); err != nil {
			panic(fmt.Sprintf("failed to load environment variables from file %s: %v", envFile, err))
		}
	}
}
```

Let me explain the main reason behind this wrapper:  
Here, we can check for the existence of the file, and in the future, we can add basic validation or sanity checks to ensure all `KEY=value` pairs are properly formatted. Think of it as a kind of "linting" check.